app/models/katello/event_queue.rb

```diff
@@ -0,0 +1,25 @@
+module Katello
+  class EventQueue < Katello::Model
+    self.table_name = 'katello_event_queue'
+
+    @repository_types = {}
```

Comment:
What does `repository` imply here? In our world, it can be an overloaded or confusing term.